test/sql/gh-5913-error-on-unsupported-type.result

```diff
@@ -0,0 +1,87 @@
+-- test-run result file version 2
```

Comment:
This is very personal, and not obligatory, but I believe it's better to create new tests inside of test/sql-tap than test/sql with result files. Could you please convert them to sql-tap for easier debug?

Reply:
I totally agree, rewrote to TAP